pkgs/development/libraries/capstone/default.nix

```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   configurePhase = '' patchShebangs make.sh '';
   buildPhase = '' ./make.sh '';
   installPhase = '' env PREFIX=$out ./make.sh install '';
+  
+  buildInputs = [
+    pkgconfig
```

Comment:
This is important for cross-compilation. All build-time dependencies should be specified in `nativeBuildInputs`.
For example host compiler would be in `nativeBuildInputs`, while a library linked against the program would be in `buildInputs`.